### ISSUE TITLE
Any plan to support bev seg?

### ISSUE THREAD

**Issue by rylynchen** [exiawsh/StreamPETR#58]
Hi, do you have plan to support bev seg?

**Comment by exiawsh** [comment]
@rylynchen Hi, thanks for your attention. Because PETRv2 already supports the task of bev seg, there is no relevant plan in this project yet. Sorry.

**Comment by rylynchen** [comment]
@exiawsh  Thanks your reply. So can I easy move the bev seg code from PETRv2 to StreamPETR, and make it work ?

**Comment by yingfei1016** [comment]
Yes, you can move seg code from PETRv2 to StreamPETR. However, the improvement will be little since the temporal modeling of segmentation has not been explored. Furthermore, dense BEV segmentation is not a suitable task for PETR series, since PETR series are sparse-query methods. If you aim at  research, you are likely to be criticized by reviewers.

Instead of dense BEV seg, you can modify PETR to support vector map (refer to MapTR). The sparse-query design will be more efficient.
